Pdf/PdfDocumentGenerator.cs

```diff
@@ -48,10 +48,10 @@ private static string ReplaceFileElementsWithMetaData(string templatePath, List<
             return tempHtmlFile;
         }
 
-        private static void ConvertHtmlToPdf(string modifiedHtmlFilePath, string outputFilePath)
+        private static void ConvertHtmlToPdf(string toolFolderAbsolutePath, string modifiedHtmlFilePath, string outputFilePath)
         {
             string wkHtmlToPdfPath = "cmd.exe";
-            string arguments = $"/C Tools\\wkhtmltopdf.exe \"{modifiedHtmlFilePath}\" \"{outputFilePath}\"";
+            string arguments = $"/C {toolFolderAbsolutePath}\\wkhtmltopdf.exe \"{modifiedHtmlFilePath}\" \"{outputFilePath}\"";
```

Comment:
Even wkhtmltopdf.exe can come in the path that the client sends

Reply:
included wkhtmltopdf exe file to the folder path and modified command as given below
string arguments = $"/C {toolFolderAbsolutePath} \"{modifiedHtmlFilePath}\" \"{outputFilePath}\"";